arm/Microsoft.Network/virtualNetworkGateways/readme.md

```diff
@@ -33,7 +33,7 @@ This module deploys a Virtual Network Gateway.
 | `metricsToEnable` | array | `[AllMetrics]` | `[AllMetrics]` | Optional. The name of metrics that will be streamed. |
 | `publicIpLogsToEnable` | array | `[DDoSProtectionNotifications, DDoSMitigationFlowLogs, DDoSMitigationReports]` | `[DDoSProtectionNotifications, DDoSMitigationFlowLogs, DDoSMitigationReports]` | Optional. The name of logs that will be streamed. |
 | `publicIPPrefixId` | string |  |  | Optional. Resource Id of the Public IP Prefix object. This is only needed if you want your Public IPs created in a PIP Prefix. |
-| `publicIpZones` | array | `[1]` |  | Optional. Specifies the zones of the Public IP address. |
+| `publicIpZones` | array | `[]` |  | Optional. Specifies the zones of the Public IP address. Basic IP SKU does not support Availability Zones.|
```

Comment:
If you are updating the parameter description, please also do so in the `deploy.bicep` file. In the contribution guide we also have info about a script under utilities/tools folder to generate the readme from the bicep file.

Reply:
Thank you for pointing that out, @MariusStorhaug! Should be fixed now. I have not seen any information on the tools available. Where can I find this in the contribution guide?

Reply:
Sorry, its in the wiki, but not the contribution guide.
https://github.com/Azure/ResourceModules/wiki/ModulesDesign#structure

Reply:
There is a reference here to a script. Though the link to it is wrong. Its under: utilities\tools\Set-ModuleReadMe.ps1